tests/test_external.py

```diff
@@ -0,0 +1,40 @@
+import copy
+
+import pytest
+
+from hangar import external
+
+
+
+@pytest.fixture()
+def plugin_reset():
+    external.reset_plugins()
+    yield None
+    external.reset_plugins()
+
+
+@pytest.fixture()
+def reset_preferred_plugins():
+    preferred_plugins = copy.deepcopy(external.manage_plugins._preferred_plugins)
+    yield None
+    external.manage_plugins._preferred_plugins = preferred_plugins
+    external.manage_plugins._write_preferred_plugins_config(preferred_plugins)
```

Comment:
This is a prime candidate for monkeypatching. Preferably with the `context manager` form to auto-reset past values. 

https://docs.pytest.org/en/latest/monkeypatch.html#global-patch-example-preventing-requests-from-remote-operations

Reply:
yep, will do that